Algorithms/Hard/4_MedianofTwoSortedArrays/Solution.java

```diff
@@ -0,0 +1,40 @@
+class Solution {
+    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
+        double median = 0.0;
+        int n1 = nums1.length;
+        int n2 = nums2.length;
+        int arr[] = new int[n1 + n2];
+        int i = 0, j = 0; 
+        int k = 0; 
+        while (i < n1 && j < n2) { 
```

Comment:
Keep a blank line before `while`. Try to always separate code blocks with a blank line so, please change in other places where appropriate.